alphapulldown/objects.py

```diff
@@ -151,6 +151,16 @@ def make_features(
                 fasta_file, self._uniprot_runner, msa_output_dir, use_precomputed_msa
             )
             self.feature_dict.update(pairing_results)
+        
+        # Add extra features to make it compatible with pickle features obtaiend from mmseqs2
+        template_confidence_scores = self.feature_dict.get('template_confidence_scores', None)
+        template_release_date = self.feature_dict.get('template_release_date', None)
```

Comment:
Have you tried to run the entire prediction cycle with mixed features? I am not sure AF accepts 'None' as the template_confidence_score and template_release_date

Reply:
> Have you tried to run the entire prediction cycle with mixed features? I am not sure AF accepts 'None' as the template_confidence_score and template_release_date

@DimaMolod This step is only preparing for the next block, which will create some fake values if these features are None. https://github.com/KosinskiLab/AlphaPulldown/blob/fe456e610f337838fff820d889293a2ead99ef14/alphapulldown/objects.py#L158-L163